docs/kyma/04-02-local-deployment.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploy Kyma locally
+type: Installation
+---
+
+This Deployment guide shows you how to quickly deploy Kyma locally on the MacOS, Linux, and Windows platforms. Kyma is deployed locally using a proprietary installer based on a [Kubernetes operator](https://coreos.com/operators/).
+
+**NOTE:** Local deployment using Minikube is deprecated.
+
+>**TIP:** See the [troubleshooting guide](#troubleshooting-overview) for tips.
+
+## Prerequisites
+
+<div tabs name="prerequisites" group="deploy-kyma-locally">
+  <details>
+  <summary label="k3s">
+  k3s
+  </summary>
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- [Docker](https://www.docker.com/get-started)
+- [k3s](https://k3s.io/) 
+
+</details>
+  <details>
+  <summary label="minikube-deprecated">
+  Minikube (deprecated)
+  </summary>
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- [Docker](https://www.docker.com/get-started)
+- [Minikube](https://github.com/kubernetes/minikube) 1.12.2
+
+</details>
+
+Virtualization:
+
+- [Hyperkit driver](https://minikube.sigs.k8s.io/docs/reference/drivers/hyperkit/) - MacOS only
+- [VirtualBox](https://www.virtualbox.org/) - Linux only
+
+> **NOTE**: To work with Kyma, use only the provided commands. Kyma requires a specific Minikube configuration and does not work on a basic Minikube cluster that you can start using the `minikube start` command.
+
+## Deploy Kyma
+
+Follow these instructions to deploy Kyma from a release or from sources:
+
+<div tabs name="deploy-kyma" group="deploy-kyma-locally">
+  <details>
+  <summary label="from-a-release-k3s">
+  From a release - k3s
+  </summary>
+
+  1. Provision a Kubernetes cluster on Minikube. Run:
+
+     ```bash
+     kyma provision k3s
+     ```
+
+  2. Deploy the latest Kyma release on k3s:
+     ```bash
+     kyma deploy
+     ```
+
+     >**NOTE:** If you want to deploy a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling `kyma deploy --source {KYMA_RELEASE}`.
+
+  </details>
+  <details>
+  <summary label="from-a-release-minikube">
+  From a release - Minikube (deprecated)
```

Comment:
Why to keep it at all? If so, we should also document how to do it with sources...
And the old way of installation is a fully different installation approach with a different set of components. So it will be not enough to have a (deprecated) text. SO I tend to delete it